lib/services/android-project-service.ts

```diff
@@ -104,6 +105,24 @@ export class AndroidProjectService extends projectServiceBaseLib.PlatformProject
 		return this._platformData;
 	}
 
+	private getDeviceBuildOutputPath(currentPath: string, projectData: IProjectData): string {
+		const currentPlatformData: IDictionary<any> = this.$projectDataService.getNSValue(projectData.projectDir, constants.TNS_ANDROID_RUNTIME_NAME),
+			platformVersion = currentPlatformData && currentPlatformData[constants.VERSION_STRING],
+			normalizedPath = path.join(currentPath, "debug");
+
+		if (semver.valid(platformVersion)) {
+			const gradleAndroidPluginVersion3xx = "4.0.0";
+			const normalizedPlatformVersion = `${semver.major(platformVersion)}.${semver.minor(platformVersion)}.0`;
+			if (semver.gte(normalizedPlatformVersion, gradleAndroidPluginVersion3xx)) {
```

Comment:
this logic can be simplified to 
```TypeScript
if (semver.lt(normalizedPlatformVersion, gradleAndroidPluginVersion3xx)) {
    return currentPath;
}
```

by inverting the logic you can skip the second `return` statement and just reuse the one on line 123